.github/workflows/smoke-test-release.yml

```diff
@@ -1,7 +1,5 @@
 name: Smoke test release
-on:
-  release:
-    types: [published]
+on: pull_request
```

Comment:
As mentioned, this change will need to be reverted so that the Action only runs on published releases.